src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -3272,17 +3290,28 @@ void AutomationInstrumentClipView::displayParameterName(int32_t paramID) {
 //display parameter value when it is changed
 void AutomationInstrumentClipView::displayParameterValue(int32_t knobPos) {
 
```

Comment:
whenever a new parameter value is set, the display parameter value function is called
thus, at the same time, store that value as the last pad selected's value so it can be used for re-drawing the knob position if necessary